internal/hcsoci/resources_lcow.go

```diff
@@ -120,6 +120,19 @@ func allocateLinuxResources(ctx context.Context, coi *createOptionsInternal, r *
 				// Mounts that map to a path in UVM are specified with 'sandbox://' prefix.
 				// example: sandbox:///a/dirInUvm destination:/b/dirInContainer
 				uvmPathForFile = mount.Source
+			} else if strings.HasPrefix(mount.Source, "hugepages://") {
+				// currently we only support 2M hugepage size
+				hugePageSubDirs := strings.Split(strings.TrimPrefix(mount.Source, "hugepages://"), "/")
```

Comment:
I know that we only support this for linux, but we should try to use `os.PathSeparator` instead of hard coding "/"

Reply:
os.PathSeparator will use the one for windows, as this is compiled on windows.

Reply:
oh yeah you're right :D sorry about this, forgot it was in the shim not the gcs code